app/advanced-analytics/custom-dashboards-reference.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Custom Dashboards reference"
+content_type: reference
+layout: reference
+description: |
+    Custom Dashboards allow you to create dashboards for your specific use cases.
+breadcrumbs:
+  - /advanced-analytics/
+products:
+    - advanced-analytics
+tags:
+  - custom-dashboards
+works_on:
+    - konnect
+api_specs:
+    - konnect/analytics-requests
+schema:
+    api: konnect/analytics-requests
+related_resources:
+  - text: Konnect Advanced Analytics
+    url: /advanced-analytics/
+  - text: Dev Portal analytics
+    url: /dev-portal/analytics/
+---
+
+Custom Dashboards provide a flexible way to build, organize, and manage analytical views that are tailored to your organization’s needs. This functionality is available in [{{site.konnect_short_name}} Analytics](https://cloud.konghq.com/us/analytics/summary).
+
+With [Dashboards](https://cloud.konghq.com/us/analytics/dashboards), you can create custom dashboards either from scratch or from a [template](#templates), and modify them from {{site.konnect_short_name}}.
+
+## Templates
```

Comment:
Should we say something about templates representing best practices delivered from Kong and that they act as a starting point. Would love to have a paragraph around why the reader should care about templates.